src/main/java/de/tum/in/www1/artemis/service/dto/AbstractBuildResultNotificationDTO.java

```diff
@@ -0,0 +1,25 @@
+package de.tum.in.www1.artemis.service.dto;
+
+import java.time.ZonedDateTime;
+import java.util.Optional;
+
+public abstract class AbstractBuildResultNotificationDTO {
+
+    public abstract ZonedDateTime getBuildRunDate();
+
+    public abstract Optional<String> getCommitHashFromAssignmentRepo();
+
+    public abstract Optional<String> getCommitHashFromTestsRepo();
+
+    public abstract boolean isBuildSuccessful();
+
+    // TODO: necessary?
```

Comment:
Is the `TODO` intended here? Or should this be handled before merging?

Reply:
Yes it is